lib/resty/wasmx/shm.lua

```diff
@@ -0,0 +1,471 @@
+-- vim:set ts=4 sw=4 sts=4 et:
+
+local ffi = require "ffi"
+local wasmx = require "resty.wasmx"
+
+
+local C = ffi.C
+local error = error
+local type = type
+local table_new = table.new
+local ffi_cast = ffi.cast
+local ffi_fill = ffi.fill
+local ffi_new = ffi.new
+local ffi_str = ffi.string
+local ngx_log = ngx.log
+local initialized = false
```

Comment:
Let's move `initialized` and `WASM_SHM_KEY` together away from the deref declarations.

Reply:
Good call.